src/components/text-input/default/index.njk

```diff
@@ -7,8 +7,8 @@ layout: layout-example.njk
 
 {{ govukInput({
   label: {
-    text: "Full name"
+    text: "Reference number"
```

Comment:
We've changed the default examples to be reference number, then the guidance says to disable spellcheck for reference numbers but we don't do it in the examples.

Is this intentional?

Reply:
No it was completely not intentional. Good spot – thanks Nick!